docs/metrics/customizing-the-sdk/README.md

```diff
@@ -224,6 +224,52 @@ default boundaries. This requires the use of
         new ExplicitBucketHistogramConfiguration { Boundaries = Array.Empty<double>() })
 ```
 
+### Customizing OpenTelemetry .NET SDK for Metrics
+
+#### Using Explicit Buckets in Histograms
+
+In OpenTelemetry .NET, histograms can be customized using the View API, and the Advice API allows additional flexibility for defining explicit bucket boundaries in instrumentation.
+
+
+##### View API vs. Advice API: Defining Histogram Buckets
```

Comment:
I understand where this is coming from. However, I suggest that we take a different approach, instead of having "vs." in our mind, focus on "how these things work together".

This is what I suggest:

1. The actual explicit buckets will be determined by the SDK based on the following factors:
    * The explicit buckets that the instrumentation owner provided via the Advice API.
    * The explicit buckets provided via View API during SDK initialization.
    * The default buckets provided by the SDK.
2. Here is how the SDK will determine the buckets:
    * ...